backend/repository/data/handler/test_cases.json

```diff
@@ -1141,23 +1256,19 @@
 								"args_spec": [
 									{
 										"id": 1,
+										"line": 100,
 										"name": "arg",
 										"required": true,
 										"type": {
-											"list_value_type": {
```

Comment:
Does the frontend UI support rendering lists without `list_value_type` ? Do you parse `type.name` instead ?